.travis.yml

```diff
@@ -1,2 +1,8 @@
 script:
-- echo "hello world"
+- docker login --username devopscorndel --password $dockerhubpass
```

Comment:
We should ideally action this step using the `--password-stdin` flag rather than passing the password explicitly, to avoid leaking those credentials to shell logs. [See the docs](https://docs.docker.com/engine/reference/commandline/login/) for more info, example of how that might look below:
```
echo $PASSWORD | docker login -u $USER --password-stdin
```